mkdocs.yml

```
@@ -0,0 +1,38 @@
+      handlers:
+        python:
+          options:
+            show_source: true
+
+# The 'nav' section defines the site's navigation.
+nav:
+  - 'Introduction': 'index.md'
+  - 'Tutorials': 'tutorials/index.md'
+  - 'How-To Guides': 'how-to/index.md'
+  - 'API Reference':
+    - 'reference/index.md'
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)